src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -31,126 +17,52 @@ export type RichTextEditorProps = {
     clear?: boolean;
 };
 
-export type BlockElement = {
-    type: BlockStyleTypes;
-    url?: string;
-    properties?: {
-        textAlign?: TextAlignTypes;
-    };
-    children: (FormattedText | BlockElement)[];
-};
-
-export type FormattedText = {
-    text: string;
-} & {
-    [key in InlineStyles]?: true;
-};
-
-declare module "slate" {
-    interface CustomTypes {
-        Editor: BaseEditor & ReactEditor;
-        Element: BlockElement;
-        Text: FormattedText;
-    }
-}
-
-const TOOLBAR_DELAY_IN_MS = 200;
 export const ON_SAVE_DELAY_IN_MS = 500;
-const isModifyingKey = (key: string) => !["Alt", "Control", "Meta", "Shift"].includes(key);
 
 export const RichTextEditor: FC<RichTextEditorProps> = ({
     value: initialValue,
     placeholder = "",
     readonly = false,
     clear = false,
     onTextChange,
-    onBlur,
```

Comment:
I'd keep the onBlur event, so we can persist the content when leaving the editor

Reply:
Makes sense, added.